.github/workflows/clean.yml

```diff
@@ -0,0 +1,39 @@
+name: clean
+
+on:
+  pull_request:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  clean:
```

Comment:
This might be better suited for `ci-core` tbh, i dont see why this should be a separate workflow, wdyt?

Reply:
How do you propose combining them? I don't think we want to add five minutes of `go generate` time to the critical path.

Reply:
you can (and should) still have it as a separate job to the core ci workflow, then it wont add any time to the tests themselves

Reply:
What is the advantage of combining of them? Am I overlooking a simplification?
They have different triggers, so it would create more noise to runs this all the time.

Reply:
Done - I guess nbd on extra runs.